packages/terminal/src/node/terminal-backend-contribution.slow-spec.ts

```diff
@@ -5,6 +5,9 @@
  * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
  */
 
+
+import 'mocha';
```

Comment:
Was this intended? Isn't the goal to replace `mocha` here with `jest`?

Reply:
These spec files are left untouched or broken (at least, I intended to) because I couldn't figure out how can I test it.